orc8r/cloud/go/models/swagger-common.yml

```diff
@@ -121,6 +121,10 @@ definitions:
     description: The type of the network
     type: string
     example: LTE
+  next_page_token:
```

Comment:
"next" page token is just a page token, so no need to include the `next_`

Reply:
was following convention in subscriber api
https://github.com/magma/magma/pull/5381/files#diff-68151344ac75147d32963d9707aa36d7b701da45e0c65b90a8931aa4d7975249R378
I'm fine with removing the next prefix but we should be consistent in our api design

Reply:
So there's a difference between the an object's field name vs. an object definition

When we're defining what a page token is (object definition), it's irrelevant whether it's the "next" page token. When we're adding a "next page token" field to e.g. a return object (object's field name), then it is indeed relevant that the page token is the "next" one

The line you linked is the field name, while the line I commented on here is the object definition